### PR TITLE
To do list:Add and Remove 

### PR DESCRIPTION
**Pull request by chiedozie013**
In this project, I:
- [ ] Removed all hardcoded items from the tasks array in index.js,
- [ ] Created a new JavaScript file for the new functionality,
- [ ] Implemented a function for adding a new task (add a new element to the array),
- [ ] Implemented a function for deleting a task (remove an element from the array),
- [ ] Implemented a function for editing task descriptions,
- [ ] Ensured all linters error passes.

### CODE REVIEW

File: modules/addRemove.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+  // Submit the form
+  form.addEventListener('submit', getList);
+  enterBtn.addEventListener('click', getList);
+
+  // Function for deleting the button
+  const removeListIndex = (index) => {
+    getStorage = getStorage.filter((item) => item.index !== Number(index));
+    displayList();
+    setStorage();
+  };
```
Comment (mikethreels) [inline modules/addRemove.js:59]: - [ ] when clicking on the trashcan Icon the item doesn't actually get deleted, please fix this.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi James, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- good job implementing the add and edit functionality.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/addRemove.js (hunk 1 above)
Comment (chiedozie013) [reply modules/addRemove.js:59]: Please can you assist me? can't seem to debug it

**Review by Owusu-Desmond — APPROVED** [review]
Hi @chiedozie013 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**